parsers_sll_ll.csv

```diff
@@ -0,0 +1,345 @@
+name,num_files,bytes,separate_run_millis
```

Comment:
What kind of file is this?

Reply:
I think this file contains some performance timings for how long it took to parse each case in SLL/LL mode, and the sizes in bytes.